package.json

```diff
@@ -32,6 +32,7 @@
   "license": "MIT",
   "jest": {
     "testURL": "http://localhost:3000",
+    "testEnvironment": "node",
```

Comment:
Test in the node environment to catch cases where we might depend on `window` without a fallback for node (using `global`)